chapters/hpmor-chapter-013.tex

```diff
@@ -171,8 +171,7 @@ \chapter{Asking the Wrong Questions}
 Then he pulled out the drawer.
 \begin{align*}\intertext{\scshape \centering
 Opportunities to do good are everywhere\\
-but darkness is where the light needs to be}
-\hbox{\scshape Cost of question: }&\hbox{\scshape 1 point}\\
+but darkness is where the light needs to be} \hbox{\scshape Cost of question: }&\hbox{\scshape 1 point}\\
```

Comment:
Please revert this, I think the markup is clearer in the original.

Reply:
done